src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,15 +1,37 @@
 package core.mate.academy.model;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Truck
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+
+    private int capacity;
+
     public Truck() {
     }
 
+    public Truck(int capacity) {
+        this.capacity = capacity;
+    }
+
     @Override
     public void doWork() {
         System.out.println("Truck started to work");
     }
+
+    @Override
+    public List<? extends Machine> get() {
```

Comment:
This method should not be here because this class should not implement interface "MachineProducer". Create class TruckProducer for it